src/dnx.host/RuntimeBootstrapper.cs

```diff
@@ -58,90 +58,97 @@ private static void PrintErrors(Exception ex)
 
         public static Task<int> ExecuteAsync(string[] args, FrameworkName targetFramework)
         {
-            var app = new CommandLineApplication(throwOnUnexpectedArg: false);
```

Comment:
It is related to aspnet/dnx#2235 and https://github.com/aspnet/Helios/issues/180